tree/ntuple/v7/inc/ROOT/RNTupleView.hxx

```diff
@@ -337,6 +337,17 @@ private:
       return RNTupleCollectionView(fieldId, fieldName, source);
    }
 
+   DescriptorId_t RetrieveFieldId(std::string_view fieldName)
```

Comment:
What's the rationale behind using `Retrieve` instead of `Get`? Is it the fact that this method can throw?